### ISSUE TITLE
Как настроить bem-grid и использовать через node_modules

### ISSUE THREAD

**Issue by exctac** [bem-site/bem-forum-content-ru#1327]
Добрый день. Заразился БЭМ) пока только пробую и учусь.

Хочу потестить bem-grid но без project-stub и собственно без полноценной БЭМ разработки,  по простому так сказать.
Установил новый модуль себе в проект: npm i --save-dev bem-grid. Подтягиваю стили из модуля, все нормально работает)
Решил поменять настройки сетки, но не пойму где именно и как пересобрать новый css, какую комманду в консоли ввести? Спасибо

**Comment by awinogradov** [comment]
1. Сетка настраивается только через сборку
1.а) Через параметры для postcss плагина
2.б) Через параметры для enb технологии

Из коробки едет собранный css c дефолтными настройками.

**Comment by exctac** [comment]
а можете направить меня где что почитать что бы выполнить правильно эти шаги?

**Comment by awinogradov** [comment]
Конечно, держи доку: https://github.com/theprotein/sharps#usage и пример для enb: https://github.com/theprotein/sharps/blob/master/.enb/make.js#L21

**Comment by exctac** [comment]
Спасибо. буду разбираться)
